README.md

```diff
@@ -92,3 +92,14 @@ A [token-bucket](https://en.wikipedia.org/wiki/Token_bucket) rate limiter implem
 
   limiter.take # true
 ```
+
+# Configuration
+
+We use a connection pool, because a connection shared among multiple threads
+will produce unexpected results when using blocking Redis operations.
+
+`RedisLocks.redis=` and the `redis:` option in the initializer for each utility
+can be passed either a plain Redis object (in which case we will construct
+a [ConnectionPool](https://github.com/mperham/connection_pool) ourselves),
+or an object responding to `with` (which will be expected to accept a block
+and yield a Redis connection).
```

Comment:
IMO you could remove the first parenthetical (since it's an implementation detail) and add the end of the sentence: "such as an instance of [ConnectionPool](https://github.com/mperham/connection_pool)," because people may not recognize the interface immediately.